File: PRACTICA#2/COMPOSICION-AGREGACION/3/PYTHON/Parte.py
class Parte:
    def __init__(self, nombre, peso):
        self.__nombre = nombre
        self.__peso = peso

    #A
    def get_nombre(self):
        return self.__nombre

    def set_nombre(self, nombre):
        self.__nombre = nombre

    def get_peso(self):
        return self.__peso

    def set_peso(self, peso):
        self.__peso = peso

    def mostrar_info(self):
        print(f"Nombre: {self.__nombre}, Peso: {self.__peso} kg")

File: PRACTICA#2/COMPOSICION-AGREGACION/3/JAVA/Parte.java
public class Parte {
    private String nombre;
    private float peso;

    public Parte(String nombre, float peso) {
        this.nombre = nombre;
        this.peso = peso;
    }

    //A
    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public float getPeso() {
        return peso;
    }

    public void setPeso(float peso) {
        this.peso = peso;
    }
    public void mostrar_info() {
        System.out.println("Nombre: " + nombre + ", Peso: " + peso + " kg");
    }
}